### PR TITLE
feat: Add support to skip users

### PR DESCRIPTION
**Pull request by bombsimon**
Add a new query parameter `skip` that can be specified multiple times to skip specific users. Can be used to filter out bots or other users that don't want to be a part of the contributors list.

### CODE REVIEW

File: src/server/lib/contributors.js [hunk 1]
```diff
@@ -32,13 +32,18 @@ export async function fetchContributors({ collectiveSlug }) {
   // Fetch data from Open Collective API
   const query = gql`
     query GithubContributors($collectiveSlug: String) {
-      Collective(slug: $collectiveSlug) {
+      collective(slug: $collectiveSlug) {
```
Comment (bombsimon) [inline src/server/lib/contributors.js]: I'm not sure if this is relevant here but due to https://github.com/opencollective/opencollective/issues/6899 I couldn't test this towards the dev API. Instead I ran it against both `https://api.opencollective.com/graphql` and `https://api.opencollective.com/graphql/v2` and both of them defines this method as lowercase `collective`.

Similarly there's no field named `githubContributors` but a `contributors` witch contains contributor nodes.

I can revert these changes that's not related to `banner.js` if you can help me confirm it's not needed.

Comment (bombsimon) [reply src/server/lib/contributors.js]: Even though I never really got the dev API working I was browsing around the code. Seems like the [local V1 is using `Collective`](https://github.com/opencollective/opencollective-api/blob/bb2eae36c87cc400c7099509fd5149a85aa026fe/server/graphql/schemaV1.graphql#L4301-L4309) but neither ones posted above or V2 in the dev API.

Comment (Betree) [reply src/server/lib/contributors.js]: https://api.opencollective.com/graphql is defaulting to https://api.opencollective.com/graphql/v2, they're indeed the same thing. The legacy API is available as https://api.opencollective.com/graphql/v1 and the field is indeed exposed as `Collective` there.

Comment (Betree) [reply src/server/lib/contributors.js]: The `contributors` field is different: it's meant to surface your financial contributors on Open Collective, while `githubContributors` is the list of contributors to your repository. I suggest rolling back that part. I can then test and merge your changes.

**Review by Betree — COMMENTED** [review]
Thank you for opening this PR! I’ve added some comments above, but the feature proposal is welcome. Out of curiosity, where did this need arise from? We’re actively working on combating spam and handling questionable contributors on the platform, so your feedback is valuable.

**Comment by bombsimon** [comment]
Thanks for the review!

> Out of curiosity, where did this need arise from? 

We were simplifying the contributions list in https://github.com/golangci/golangci-lint/pull/5148 and wanted to use this feature to make the process simpler. The process we had already supported filter out contributors but mainly to remove bots and non-human contributors.

In https://github.com/golangci/golangci-lint/pull/2974 we were asked to also remove a specific individual from our contributors list which was merged.

Since we now have this image in our readme I was looking for a simple way to support this. Mainly with respect to the ask we got, but also to remove bots since we have a few of them with many automated contributions.

File: src/server/lib/contributors.js (hunk 1 above)
Comment (bombsimon) [reply src/server/lib/contributors.js]: I reverted these changes! If it's not too much effort, do you mind a quick explanation on how you test this? As mentioned I had to create a real key and use the public API to test this but obviously I tested `v2` two times by mistake 🙈 

I opted to setup the test API with PostgreSQL 13 which is latest version I got working but it didn't work out of the box even though I was ensuring I was setting the right key that was set in the `.env`. Also is there another way other than querying the db for collectives to see what data is in the dump to test things like this? Something like a GraphQL Playground/Grapiql?

**Pull request merged**